src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -79,42 +108,23 @@
       return input.getString();
     }};
 
-  private List<JSError> errors = new ArrayList<>();
-  private StringWriter out = new StringWriter();
   private final Options opts;
-  private Set<String> privateEnums = new HashSet<>();
+  private final Compiler compiler;
+  private final ClutzErrorManager errorManager;
+  private StringWriter out = new StringWriter();
+  private final Set<String> privateEnums = new HashSet<>();
 
   DeclarationGenerator(Options opts) {
     this.opts = opts;
+    this. compiler = new Compiler();
```

Comment:
The problem is that the Eclipse formatter would wreck the whole file if run on it, and there's no good option to only run it on changed lines. I wish google-java-format had an Eclipse plugin already...